docs/gettingstarted.rst

```diff
@@ -150,10 +150,13 @@ Deploy on CERN infrastructure
       # Get the Kube master name and connect to it
       $ openstack server list | grep -E reana-.*-master
       $ ssh -i <ssh-key> fedora@<master-node>
-      # Add to the `--feature-gates` the `TTLAfterFinished=true` flag
+      # Add to the `--feature-gates` the `TTLAfterFinished=true` flag to the
```

Comment:
I would rephrase it to ``# Add `TTLAfterFinished=true` to the `--feature-gates` in `/etc/kubernetes/apiserver `and  `/etc/kubernetes/controller-manager` `` or something similar. I think it would be more clear. What do you think?